docs/section19/Rule19-13.md

```diff
@@ -72,9 +72,9 @@ Exceptions:
         
         
         - **Rule Assertion:** 
-        - Case 1: If the supply-air-to-room temperature set-point difference equals 20°F or 17F per G3.1.2.8.1 including Exceptions 1, the fan is autosized and supply cfm is greater or equal to the min req OA cfm then pass: `if all_design_setpoints_delta_Ts_are_per_reqs == true and are_all_hvac_sys_fan_objs_autosized(B_RMI, hvac) == True and list(get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM(B_RMI, hvac.fan_system)["Fan_CFM"])[0] >= hvac.fan_system.minimum_outdoor_airflow: outcome = "PASS"`  
-        - Case 2: Else if the supply-air-to-room temperature set-point difference does not equal 20°F or 17F per G3.1.2.8.1 including Exceptions 1, and the fan system supply cfm equals the fan system minumum OA cfm (i.e., airflow rate is minimum outdoor airflow rate) then pass: `elif all_design_setpoints_delta_Ts_are_per_reqs == false and list(get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM(B_RMI, hvac.fan_system)["Fan_CFM"])[0] == hvac.fan_system.Minimum_Outdoor_Airflow: outcome = "PASS"`  
-        - Case 3: Else if the supply-air-to-room temperature set-point difference does not equal 20°F or 17F per G3.1.2.8.1 including Exceptions 1, and the baseline fan system supply cfm equals the proposed (i.e., airflow rate could be dictated by applicable codes or accreditation standard) then UNDETERMINED: `elif all_design_setpoints_delta_Ts_are_per_reqs == false and list(get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM(B_RMI, hvac.fan_system)["Fan_CFM"])[0] == proposed_supply_cfm: outcome = "UNDETERMINED" and raise_message "<insert hvac.id> was not modeled based on a supply-air-to-room temperature set-point difference of 20°F (or 17°F, if lab). The baseline and proposed supply cfm was modeled identically at <insert proposed_supply_cfm> CFM. Manual review is required to determine if the airflow rate was modeled to comply with applicable codes or accreditation standards. If not, fail."`   
```

Comment:
@yunjoonjung-PNNL I removed the list() around the code because it is no longer a list (its a single value) so I don't think its needed. Please let me know if I am mistaken.

Reply:
Looks great to me!